src/components/Orders.js

```diff
@@ -27,6 +27,7 @@ class Orders extends Component {
 
   componentWillMount() {
     this.props.loadOrders();
+    this.props.loadItems();
```

Comment:
moving this to the orders page. I feel like we should have a getInitial state action that get's called the first time the app loads. What do you think?